src/core/core.js

```diff
@@ -452,4 +454,87 @@ export default class Core {
   on (evt, moduleId, cb) {
     return this.globalStorage.on(evt, moduleId, cb);
   }
+
+  getUrlParams () {
```

Comment:
it /feels/ to me like we need our own `ui/tools/urlutils.js` for these functions. it doesnt feel like any of this code is related to the core itself (it references UI classes, is only used in UI components).

i've been thinking about this further and it seems like most of our new `core/utils/urlutils` (that deal with search params) should belong in the `ui` folder and not the `core` folder.

what do you think?